db/sql/voters/get_voters.sql

```diff
@@ -19,7 +19,8 @@
   PARAMETERS: ?
 */
 
-SELECT "accountId"
-FROM mem_accounts2delegates
-WHERE "dependentId" = ${publicKey}
+SELECT t2.address, t2.balance, encode(t2."publicKey", 'hex') AS "publicKey" FROM mem_accounts2delegates t1
+INNER JOIN mem_accounts t2 ON t1."accountId" = t2.address
```

Comment:
For better readability, please rename `t2` to `accounts` and `t1` to `delegates`

Reply:
@nazarhussain done.